frontend/src/components/ViewTrips/save-trip-modal.js

```diff
@@ -48,7 +74,7 @@ class SaveTripModal extends React.Component {
     this.isAddTripForm = this.props.tripId === null;
 
     // For edit trips, create the number of collaborator input box refs as one
-    // less than the number of collaborators specified in prop `defaultFormObj`
+    // less than the number of collaborators specified in prop `defaultFormData`
     // (do not include current user in list).
     //
```

Comment:
```suggestion
    
```

blank line

Reply:
I am going to change this to a block comment because I like the line separation between these two sentences but it does look a little weird with these types of comments.